hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -380,7 +380,7 @@ private OmMetadataManagerImpl(OzoneConfiguration conf, File dir, String name)
     omEpoch = 0;
     setStore(loadDB(conf, dir, name, true,
         java.util.Optional.of(Boolean.TRUE), Optional.empty()));
-    initializeOmTables(false);
+    initializeOmTables(CacheType.FULL_CACHE, false);
```

Comment:
This code path is used by Bootstrapping code. I don't think we need to load the cache in this case. Same for other config like `enableCompactionDag` and `maxOpenFiles`. Please revisit them as well.

Reply:
created follow up task(HDDS-9781) to revisit order configs.